mocha-phantomjs-core.js

```diff
@@ -114,6 +114,15 @@ page.onCallback = function(data) {
       output.write(data.stdout)
     } else if (typeof data.screenshot === 'string') {
       page.render(data.screenshot + '.png')
+    } else if (data.viewportSize && (data.viewportSize.x || data.viewportSize.y)) {
+      setTimeout(function() {
+        page.viewportSize = (function(_page, _current, _x, _y) {
+          return _page.viewportSize = {
+            width : _x || _current.width,
+            height : _y || _current.height
+          };
+        })(page, page.viewportSize, data.viewportSize.x, data.viewportSize.y);
```

Comment:
Strange that first time I had tried it this way, it was not working - but then had fixed it with `setTimeout`.

Tried it again - and it's fine now.